### DISCUSSION TITLE
Welcome to SpaceX-prepare-for-Clean-Architecture-liftoff Discussions!

### DISCUSSION THREAD

**Discussion by ferPrieto** [ferPrieto/spaceX#92] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hariomop12** [comment]
Very nice and clear project thankyou for sharing this awesome project and clearly understandable project sir thank you 😊

**Reply by ferPrieto** [reply]
Why clearly unstable though? @hariomop12

**Reply by hariomop12** [reply]
> Why clearly unstable though? @hariomop12 

Soory sir it's understandable project it's my typo mistake 🥲

**Reply by hariomop12** [reply]
> Why clearly unstable though? @hariomop12 

I learn many things from this project

**Reply by ferPrieto** [reply]
Ah ok, I got confused 😄 There will be many updates and new content online soon. Stay tuned! :octocat:

**Reply by hariomop12** [reply]
tnx sir i  am waiting
